index.js

```diff
@@ -27,7 +27,29 @@ module.exports = class NitroBypass extends Plugin {
 		}, 1000);
 
 		const emojiReplacePatch = this.emojiReplacePatch.bind(this);
-		inject('replace-on-send', message, 'sendMessage', emojiReplacePatch, true);
+		const stickerReplacePatch = this.stickerReplacePatch.bind(this);
+		const supressClyde = this.supressClyde.bind(this);
+		
+		inject('replace-on-send', this.message, 'sendMessage', emojiReplacePatch, true);
+		inject('supress-clyde', this.message, 'sendClydeError', supressClyde, true);
```

Comment:
This would completely disable Clyde, we shouldn't patch Clyde completely

Reply:
In the suppressClyde function i check the second arg which I assume is the error id and don't let only the ones with the id that I logged trough. I haven't tested it though.